ONSdigital/census-rm-acceptance-tests-2021
# Ensure Action Cancel is not sent to Field from continuation questionnaire receipts

# Motivation and Context

When we receive a PQRS receipt for a continuation form we need ensure that fieldwork are not sent an Action Cancel instruction from this so that follow-ups can continue until the main form is receipted. 

# What has changed



* Added two new tests for continuation receipts from an unaddressed scenario and from a fulfilment request scenario
* Re-jigged some of the steps to reuse existing functions

# How to test?



Checkout the fieldwork-adaptor PR from link below and run these tests to ensure no Action Instruction messages are sent to field from these tests.

# Links



https://trello.com/c/cvCv6U1R/499-do-not-send-a-cancel-message-when-a-continuation-form-is-completed-3
https://github.com/ONSdigital/census-rm-fieldwork-adapter/pull/26
